lib/simp/beaker_helpers.rb

```diff
@@ -374,17 +382,17 @@ def enable_fips_mode_on( suts = hosts )
         on(sut, module_install_cmd)
       end
 
+      # Needed because various versions of crypto-policies are broken
+      on(sut, "puppet resource package crypto-policies ensure=latest") if has_crypto_policies(sut)
+
       # Enable FIPS and then reboot to finish.
       on(sut, %(puppet apply --verbose #{fips_enable_modulepath} -e "class { 'fips': enabled => true }"))
 
       # Work around Vagrant and cipher restrictions in EL8+
       #
       # Hopefully, Vagrant will update the used ciphers at some point but who
       # knows when that will be
-      opensshserver_config = '/etc/crypto-policies/back-ends/opensshserver.config'
-      if file_exists_on(sut, opensshserver_config)
-        on(sut, "sed --follow-symlinks -i 's/PubkeyAcceptedKeyTypes=/PubkeyAcceptedKeyTypes=ssh-rsa,/' #{opensshserver_config}")
-      end
+      munge_ssh_crypto_policies(sut)
```

Comment:
this should only be done if `has_crypto_policies(sut)` returns `true`

Reply:
The operations ignore failures. Not the best practice but it also doesn't matter if the commands fail for some reason.

I can try for consolidation if I push up the package update.